frontend/src/pages/Home/components/JoinUsForm/Form.jsx

```diff
@@ -67,311 +202,318 @@ export const JoinUsForm = (props) => {
                   }
                 >
                   <input
-                    placeholder="Contact No."
-                    id="phone"
-                    type="tel"
+                    placeholder="Linkedin Profile URL"
+                    id="txt_link"
+                    type="text"
                     required="required"
```

Comment:
same as above(no need)

Reply:
actually these were already present in the file before I did anything

Anyways , I will remove them , no issues